packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -678,17 +678,15 @@ export class EngineReportingAgent<TContext = any> {
     executableSchemaId: string;
     executableSchema: string;
   }) {
-    this.logger.info('Starting schema reporter...')
-
-    this.logger.info(
-      `Schema reporter options: ${JSON.stringify(
-        {
-          overrideReportedSchema: this.options.experimental_overrideReportedSchema,
-          schemaReportingInitialDelayMaxMs: this.options.experimental_schemaReportingInitialDelayMaxMs,
-          schemaReportingUrl: this.options.schemaReportingUrl,
-        }
-      )}`
-    )
+    this.logger.info('Starting schema reporter...');
+    this.logger.info(`Schema reporting override is ${
+      this.options.experimental_overrideReportedSchema ?
```

Comment:
This line only gets if we decide to start reporting, so it must always be true. I think we can get rid of this line. It also doesn't handle the case that someone overrides with an environment variable so they could see that `schemaReporting` is disabled but still be schema reporting and be kind of confused by that.

Reply:
Are you thinking of `this.options.experimental_schemaReporting` instead of `this.options.experimental_overrideReportedSchema`? I can change this line to instead print `Schema to report has been overridden` when `this.options.experimental_overrideReportedSchema` is not undefined if it's clearer.

Reply:
Misread the line yea I was thinking of `this.options.experimental_schemaReporting`.